knowledge-system/scripts/start_cks_simple.sh

```
@@ -0,0 +1,69 @@
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+NC='\033[0m'
+
+echo -e "${YELLOW}Starting CKS API Server...${NC}"
+
+# Kill any existing process on port 5000
+lsof -ti:$API_PORT | xargs kill -9 2>/dev/null
+
+# Start the API server
+cd "$CKS_ROOT"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)